content/docs/reactivesearch/v3/search/datasearch.md

```diff
@@ -105,6 +105,8 @@ Example uses:
     > Note:
     >
     > Popular Suggestions only work when `enableAppbase` prop is `true`.
+-   **enablePredictiveSuggestions** `bool` [optional]
```

Comment:
@lakhansamani Can you move that prop after `enablePopularSuggestions`?
Make that change for vue too.
Also, add docs for category search.

Reply:
@bietkul its already after `enablePopularSuggestions` for all the three components
its just that there is note attached to `enablePopularSuggestions`

[](https://www.dropbox.com/s/q2cfx9e0if9f0ys/Screenshot%202020-12-23%20at%2011.07.57%20PM.png?raw=1)